packages/react-aria-components/src/Slider.tsx

```diff
@@ -21,6 +21,11 @@ export interface SliderProps<T = number | number[]> extends Omit<AriaSliderProps
   /**
    * The display format of the value label.
    */
+  format?: Intl.NumberFormatOptions | ((value: number, locale: string) => string),
+  /**
+   * The display format of the value label.
+   * @deprecated Use `format` instead.
```

Comment:
> I'll see if anyone else on the team has thoughts.

Any updates on this?

Reply:
Apologies, I let it slip. I've posted it now and will hopefully have an update soon.

Reply:
Yeah, I think it would be good to maintain consistency with our other components that also use formatOptions. That said, I do think it would be good to support this so not really sure how we would go about this without making changes to formatOptions.

Reply:
Thanks for your patience, I'm bringing this up again.